CMakeLists.txt

```diff
@@ -295,6 +297,10 @@ else()
     set(DLT_JSON_LIBRARY "")
 endif()
 
+if (WITH_DLT_LOG_LEVEL_APP_CONFIG)
```

Comment:
it is correct with WITH_LOG_LEVEL_APP_CONFIG

Reply:
We agreed on WITH_DLT_LOG_LEVEL_APP_CONFIG